util/reggen/field.py

```diff
@@ -180,6 +204,14 @@ def from_raw(reg_name: str,
 
         return Field(name, desc, tags, swaccess, hwaccess, bits, resval, enum)
 
+    @staticmethod
+    def get_mubi_val(mubi_type: str, mubi_val: bool) -> str:
+        mubi_val_true, mubi_val_false = all_mubi_values()
```

Comment:
can we maybe call `mubi_value()` directly here? we technically only need to construct one specific value and not the entire list...

Reply:
i forgot to remove this, we don't need it anymore.